### DISCUSSION TITLE
storage_options arg in to_zarr?

### DISCUSSION THREAD

**Discussion by raybellwaves** [pydata/xarray#5601] [Ideas] [answered]
Is there a `storage_options` arg in [`to_zarr`](http://xarray.pydata.org/en/stable/generated/xarray.Dataset.to_zarr.html) similar to [`dask.array.to_zarr`](https://docs.dask.org/en/latest/array-api.html#dask.array.to_zarr)?

I know [`open_dataset`](http://xarray.pydata.org/en/stable/generated/xarray.open_dataset.html) can make use of `backend_kwargs=dict(storage_options=storage_options)`.

cc. @martindurant

**Comment by martindurant** [comment]
It seems that this still works by having to create your own mapping store (`fs.get_mapper`) as a separate step. Adding `storage_options`, as with the read side, would be very reasonable.

**Comment by raybellwaves** [comment, marked as answer]
closed via https://github.com/pydata/xarray/pull/5615
